server/cmd/dcrdex/main.go

```diff
@@ -30,6 +30,20 @@ func mainCore(ctx context.Context) error {
 		fmt.Printf("Failed to load dcrdata config: %s\n", err.Error())
 		return err
 	}
+	// Load asset driver
+	// cfg.AssetsDataDir
+	files, err := ioutil.ReadDir(cfg.AssetsDataDir)
+	if err != nil {
+		fmt.Printf("Failed to load asset plugin: %s\n", err)
+		return err
+	}
+
+	for _, file := range files {
+		if err == nil && !file.IsDir() && filepath.Ext(file.Name()) == ".so" {
+			plugin.Open(filepath.Join(cfg.AssetsDataDir, file.Name()))
```

Comment:
Need to check error return form `Open`.  There are a lot of ways opening a dynamic lib can fail.